cpanfile

```diff
@@ -84,6 +84,10 @@ requires 'Log::Any::Adapter::Log4perl', '>= 0.09'; # liblog-any-adapter-log4perl
 requires 'Action::CircuitBreaker';
 requires 'Action::Retry'; # deps: libmath-fibonacci-perl
 
+# AnyEvent
```

Comment:
what about the EV module?

Reply:
AnyEvent brings the INotify module. There is no direct integration with EV.

AnyEvent is an abstraction, EV is the event implementation layer. So we need both.